### ISSUE TITLE
Cant change user to root / cant install anything

### ISSUE THREAD

**Issue by idanmiller97** [liquibase/docker#173]
Hi, after I pulled liquibase docker, im trying to install for example AWS cli.
When I was trying to install the tool, im getting this error : "mkdir: cannot create directory ‘/usr/local/aws-cli’: Permission denied"
I saw that liquibase doesn't have permissions to create files and I cant run sudo to root.
What can be done ? thanks !

**Comment by jnewton03** [comment]
Hi @idanmiller97 , you can extend the Dockerfile to create your own custom image that includes the AWS CLI like so:
```
FROM liquibase/liquibase:latest
USER root
RUN wget "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -O "awscliv2.zip" && \
unzip awscliv2.zip && rm -rf awscliv2.zip && \
./aws/install
USER liquibase
```

**Comment by jnewton03** [comment]
Fixed in #174
